crates/floresta-chain/src/pruned_utreexo/consensus.rs

```diff
@@ -212,26 +213,19 @@ impl Consensus {
     /// Consumes the UTXOs from the hashmap, and returns the value of the consumed UTXOs.
     /// If we do not find the UTXO, we return an error invalidating the input that tried to
     /// consume that UTXO.
```

Comment:
Could you update the docstring ?

something like

```Rust
/// Returns the amout of satoshis of the utxo being spent in the given input.
///
/// Fails if the utxo being spent is not present in the given hashmap.
```

Just because now is not consuming anything, just checking its validity.

Reply:
Oh yeah I forgot the comment